MWSE/TES3Birthsign.h

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+#include "TES3Defines.h"
+
+#include "TES3Object.h"
+#include "TES3SpellList.h"
+
+namespace TES3 {
+	struct Birthsign : BaseObject {
+		char id[32];						// 0x10
+		char * name;						// 0x30
+		char * texturePath;					// 0x34
+		char * description;					// 0x38
+		TES3::SpellList spellList;			// 0x3C
+		unsigned int descriptionFileOffset; // 0x54
+		
+		char * getName();
+		char * getDescription();
+		void loadDescription();
```

Comment:
Is this function ever implemented? It doesn't need to be, and can be removed from the header.